src/main/boilerplate/spray/json/ProductFormatsInstances.scala.template

```diff
@@ -16,14 +16,21 @@
 
 package spray.json
 
+import scala.collection.mutable.HashSet
+
 trait ProductFormatsInstances { self: ProductFormats with StandardFormats =>
+  def allowExtraKeys: Boolean = true
```

Comment:
I wonder (but I'm not sure) if reversing the naming could be more clear? `failOnExtraKeys` or something like this?